topgun/k8s/k8s_suite_test.go

```diff
@@ -139,12 +269,13 @@ func helmDeploy(releaseName, namespace, chartDir string, args ...string) *gexec.
 
 func helmInstallArgs(args ...string) []string {
 	helmArgs := []string{
-		"--set=web.livenessProbe.failureThreshold=3",
-		"--set=web.livenessProbe.initialDelaySeconds=3",
-		"--set=web.livenessProbe.periodSeconds=3",
-		"--set=web.livenessProbe.timeoutSeconds=3",
 		"--set=concourse.web.kubernetes.keepNamespaces=false",
+		"--set=concourse.worker.bindIp=0.0.0.0",
 		"--set=postgresql.persistence.enabled=false",
+		"--set=web.resources.requests.cpu=500m",
```

Comment:
by placing `requests` but not limits, we let processes that need a lot of power (in a spike during startup, e.g.) to leverage that (as there are no actual limits being placed), while still leveraging the proper scheduling from k8s.